src/assets/locales/en.js

```diff
@@ -103,6 +107,17 @@ export default {
           title: "Publishing the plio...",
         },
       },
+      image_uploader: {
+        title: "Drag and drop or click here to upload",
+        buttons: {
+          done: "Done",
+          delete: "Delete",
+        },
+        size_info_text: {
+          info: "Max image size allowed - 10 MB",
```

Comment:
based on the appropriate back translation of the hindi one
```suggestion
          info: "Maximum image size allowed is 10 MB",
```